### PR TITLE
Make console placeholder a configurable string

### PR DESCRIPTION
**Pull request by Maarten88**
As requested in #183

### CODE REVIEW

File: src/Strings.ts [hunk 1]
```diff
@@ -27,6 +28,7 @@ const localizedStrings: LocalizedStrings = {
         messageFailed: "couldn't send",
         messageSending: "sending",
         timeSent: " at %1",
+        placeholder: "Type your message..."
```
Comment (danmarshall) [inline src/Strings.ts]: perhaps we can name this more specifically - I can see a case where there may be other "placeholders"

Comment (Maarten88) [inline src/Strings.ts]: I did think about better names, but could not decide. Shall I make it consolePlaceholder? textboxPlaceholder?

Comment (danmarshall) [inline src/Strings.ts]: Yeah I hear that. How about sendTextboxPlaceholder ?

**Comment by billba** [comment]
Perfect, thanks!

**Pull request merged**